docs/operate/relayer/run-relayer.mdx

```diff
@@ -47,21 +47,69 @@ See the [Agent Keys](../set-up-agent-keys.mdx) page to set up your Hexadecimal o
+
+Cosmos-based chains require both **RPC and gRPC endpoints** for proper operation.
+
+- **Configure multiple RPCs:** Use [`customRpcUrls`](../config-reference.mdx#chainschain_namecustomrpcurls).
+- **Configure gRPC endpoints:** Use [`customGrpcUrls`].
+- **Fallback Mechanism:** Cosmos agents always use **fallback mode**.
+
+</TabItem>
+
+<TabItem value="svm" label="SVM Chains">
+
+SVM-based chains support configuring multiple RPC endpoints, but **only a single RPC is used**.
+
+- **Configure RPCs:** Use [`customRpcUrls`](../config-reference.mdx#chainschain_namecustomrpcurls).
+
+:::caution
+
+For **SVM chains**, only a **single RPC URL** is used. here is no fallback or quorum-based selection mechanism.
+
+:::
+
+</TabItem>
+
+</Tabs>
+
+#### Configuration Settings
+
 Like the local setup, there are a few base arguments you should provide when configuring your Relayer.
 
 | Argument                        | Description                                                                                                                                                                                                                                                                                                                                               |
 | ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `--relayChains`                 | Comma separated names of the origin and destination chains to relay messages between. For example: `ethereum,polygon,avalanche`.                                                                                                                                                                                                                          |
 | `--db`                          | The path to where the Relayer should write persistent data to disk. Ensure this path to be persistent when using cloud setups. When using Docker, make sure to mount the persistent path/volume into the container. See [config-reference](../config-reference.mdx#db) for more info.                                                                     |
 | `--allowLocalCheckpointSyncers` | If `true`, this will allow the Relayer to look for Validator signatures on the Relayer's local filesystem. In a production environment, this should be `false`. If you're running a Validator on the same machine by following the Validator local setup instructions, set this to `true` so that your Relayer can access the local Validator signatures. |
+| `--chains.[originChainName].customRpcUrls`      | Override the default RPC URLs used by the Validator for your origin chain.                              |
```

Comment:
nit: fwiw these aren't required and I wouldn't consider them "base" arguments, so I don't think they're totally necessary to be here if you think the above section on `RPC Configuration` is sufficient. Otherwise I'd maybe mention some more details here that these aren't required and/or link to the config reference on them